.github/workflows/migrations-check.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     steps:
     - name: Setup mongodb user
       run: |
-        mongosh edxapp --eval '
+        docker exec ${{ job.services.mongo.id }} mongosh edxapp --eval '
```

Comment:
THis commit could be a cherry-pick https://github.com/openedx/edx-platform/commit/d8eef6e347b36c61221c455fdf1931290939e298